lib/containers/common.pm

```diff
@@ -252,4 +252,20 @@ sub scc_restore_docker_image_credentials {
     assert_script_run "cp /etc/zypp/credentials.d/SCCcredentials{.bak,}" if (is_sle() && get_var('SCC_DOCKER_IMAGE'));
 }
 
+sub test_rpm_db_backend {
+    my %args     = @_;
+    my $image    = $args{image};
+    my $runtime  = $args{runtime};
+    my $expected = 'bdb';
+
+    die 'Argument $image not provided!'   unless $image;
+    die 'Argument $runtime not provided!' unless $runtime;
+
+    my $backend = script_output "$runtime run $image rpm --eval %_db_backend";
+    my ($running_version, $sp, $host_distri) = get_os_release("$runtime run $image");
+    record_info('RPM check', "The rpm db backend in the image $image is $backend");
+    $expected = 'ndb'                             if ($running_version eq '15' && $sp eq '3');
```

Comment:
Leap still uses bdb

Reply:
Good catch. Fixed and doing VR against Leap too.

Reply:
TW uses ndb, I will add that condition too.